app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -1,3 +1,5 @@
+$space-between-data: govuk-spacing(1);
```

Comment:
I'm not sure what value is added by defining this when it's only used a couple of times. I'm not going to block merging based on it though.